docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -56,13 +56,124 @@ This makes some tasks more difficult to complete interactively.
 Setting up an Interactive Sim
 -----------------------------
 
-.. todo::
+Setting up an interactive sim for the first time can be confusing. Follow these 
+instructions along with the example notebook included for help. 
+Another useful resource is the `Vivarium InteractiveContext documentation <https://vivarium.readthedocs.io/en/latest/api_reference/interface/interactive.html?highlight=InteractiveContext#vivarium.interface.interactive.InteractiveContext>`_. 
+
+**Example notebook**
+
+This `notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_sim_example_setup.ipynb>`_ includes all of the steps seen below using the CVD project as an example. 
+
+**Step 1: General Prep**
```

Comment:
Should we make these subheadings? That would make them linkable, show up in the TOC, etc